nox.py

```diff
@@ -15,26 +15,23 @@
 import nox
 
 
-@nox.session
+@nox.session(python='3.6')
```

Comment:
should we rev to 3.7 . at the same time?

Reply:
Not yet, I wanna cut over all of our tools to 3.7 when I get back from vacation.